DOCSP-44681: Client bulk write (#984)
mongodb/docs-node: source/fundamentals/crud/write-operations/bulk.txt

--------

In this guide, you can learn how to use the {+driver-short+} to
perform **bulk operations**. Bulk operations help reduce the number
of calls to the server. Instead of sending a request for each operation,
you can perform multiple operations within one action.

.. tip::

   To learn more about bulk operations, see :manual:`Bulk Write Operations </core/bulk-write-operations/>`
   in the {+mdb-server+} manual.

You can use bulk operations to perform multiple write operations on

> a collection. You can also call bulk operations on your client,

Reviewer: ```suggestion
a collection. You can also execute bulk operations from the client,
```

Author: Fixed, but used "run" to [avoid "execute"](https://www.mongodb.com/docs/meta/style-guide/terminology/terms-for-global-audience/#use-jargon-with-care)

      Modified documents: 2

.. _node-bulk-replace-client:

Client Bulk Replacements
~~~~~~~~~~~~~~~~~~~~~~~~

To perform a bulk replace operation across multiple collections or
databases, create a ``ClientBulkWriteModel`` for each operation. Then,
call the ``bulkWrite()`` method on your client and pass an array of
models as a parameter. Use the following syntax to create

> a ``ClientBulkWriteModel`` that specifies an insert operation:

Author: yes, thanks for catching!
